src/TrackPanel.cpp

```diff
@@ -1444,6 +1503,9 @@ struct Subgroup final : TrackPanelGroup {
          for ( auto channel : TrackList::Channels( leader ) ) {
             auto &view = TrackView::Get( *channel );
             height += view.GetHeight();
+
+            if (view.GetAffordanceControls())
+               height += kAffordancesAreaHeight;
```

Comment:
You have redefined subdivisions in this file.  I like that.  It would be good to make a commit just of the changes of TrackPanel.cpp, and the changes in TrackView necessary for it.  Then, other commits to implement affordance controls specially for WaveTrack and for NoteTrack.

Reply:
I have observed this:  If you minimize a Wave track (the button with a black triangle), it is taller than it was before.  "Mute" and "Solo" buttons still appear.

Is this an intended consequence?  Is it desirable?  Please point it out to the designers and ask.

Reply:
I don't remember that desiners would requested such behaviour, it rather a unforseen consequence of having an affordance area which increases minimum track size. I'll let them know about this.